src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -371,8 +371,6 @@ public List<Assessment> getAssessments(final Long bookingId) {
         return assessments;
     }
 
-    // @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "VIEW_PRISONER_DATA"})
```

Comment:
All the service calls are being called by two methods which both set the overrideRoles - so can be removed from these